src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -96,96 +69,102 @@ export class ResourceInstanceFormComponent implements OnInit, OnChanges, OnDestr
 
     propertiesObj = {};
 
-    // feature toggle for new concept
-    beta = false;
-
     constructor(
         @Inject(DspApiConnectionToken) private _dspApiConnection: KnoraApiConnection,
         private _errorHandler: ErrorHandlerService,
         private _fb: UntypedFormBuilder,
-        private _project: ProjectService,
         private _resourceService: ResourceService,
         private _route: ActivatedRoute,
         private _router: Router,
-        private _sortingService: SortingService
-    ) {
-        // get feature toggle information if url contains beta
-        if(this._route.parent) {
-            this.beta = (this._route.parent.snapshot.url[0].path === 'beta');
-        }
-    }
+        private _componentCommsService: ComponentCommunicationEventService,
```

Comment:
Shouldn't be "_componentCommService" instead of "_componentComm`s`Service"?

Reply:
I think it's fine. Technically the service should have an `s` at the end of `Communication`.